docs/src/main/tut/guidelines.md

```diff
@@ -11,17 +11,64 @@ All guidelines in cats should have clear justifications. There is no room for tr
 
 ## Syntax
 
-### Composing Implicit Conversions in Traits
+### <a id="implicit-syntax-conversions" href="#implicit-syntax-conversions"></a> Composing Implicit Conversions in Traits
 
 Implicit syntax conversions provided in publicly-exposed traits should be marked final 
 so that any composition of the traits provides conversions that can all be inlined.
 
-### Ops Classes
+### <a id="ops-classes" href="#ops-classes"></a> Ops Classes
 
 Ops classes should be marked final and extend AnyVal, to take full advantage of inlining and prevent unnecessary allocations.
 
 The most notable exception is the case where all of the ops in the class are provided by zero-cost macros anyway,
 for example with Simulacrum.
 
+### <a id="partially-applied-type" href="#partially-applied-type"></a> Partially-Applied Type
```

Comment:
I used the term "Partially-Applied Type" to refer to the intermediate class as a new type with type parameters partially applied. Now that I said it out loud, It's quite a mouthful. I'll change it.